src/sysupdate/updatectl.c

```diff
@@ -824,9 +824,9 @@ static int update_properties_changed(sd_bus_message *m, void *userdata, sd_bus_e
         Operation *op = ASSERT_PTR(userdata);
         OrderedHashmap *map = ASSERT_PTR(op->userdata);
         const char *interface;
-        unsigned progress = UINT_MAX;
+        uint32_t progress = UINT32_MAX;
         const struct bus_properties_map prop_map[] = {
```

Comment:
if you'll do it this way, you should make the table `static const` - there's nothing that can change about it at runtime